prow/plugins/cla/cla.go

```diff
@@ -58,12 +57,19 @@ It may take a couple minutes for the CLA signature to be fully registered; after
 )
 
 var (
-	checkCLARe = regexp.MustCompile(`(?mi)^/check-cla\s*$`)
+	checkCLARe     = regexp.MustCompile(`(?mi)^/check-cla\s*$`)
+	claContextName string
 )
 
 func init() {
 	plugins.RegisterStatusEventHandler(pluginName, handleStatusEvent, helpProvider)
 	plugins.RegisterGenericCommentHandler(pluginName, handleCommentEvent, helpProvider)
+	config := &plugins.Configuration{}
+	if config.EasyCLAEnabled {
+		claContextName = "cla/easy-cla"
+	} else {
+		claContextName = "cla/linuxfoundation"
+	}
```

Comment:
If I'm not wrong, this will always set the `claContextName` to `cla/linuxfoundation` because `EasyCLAEnabled` will default to `false` in an empty `Configuration` struct?

Reply:
Ahhh...right. Thanks!